misc/config_tools/schema/checks/passthrough_devices.xsd

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<xs:schema xml:id="root"
+	   xmlns:xs="http://www.w3.org/2001/XMLSchema"
+	   xmlns:acrn="https://projectacrn.org">
+
+  <xs:assert test="every $vm in /acrn-config/vm satisfies
+                   starts-with($vm//vm_type, 'PRE') or not($vm//mmio_resources/p2sb = 'y')">
+    <xs:annotation acrn:severity="error">
+      <xs:documentation>VM {$vm/@id} is not a pre-launched VM and cannot have a P2SB (Primary-to-Sideband bridge) passed through to it.  Remove the mmio_resources.p2sb value for this VM.</xs:documentation>
+    </xs:annotation>
+  </xs:assert>
+
+  <xs:assert test="every $vm in /acrn-config/vm satisfies
+                   not($vm//mmio_resources/TPM2 = 'y') or not($vm//mmio_resources/p2sb = 'y')">
+    <xs:annotation acrn:severity="error">
+      <xs:documentation>VM {$vm/@id} is assigned both a TPM2 and P2SB (Primary-to-Sideband bridge), which is not a supported configuration. Remove one of these choices.</xs:documentation>
```

Comment:
Same comment here (show the name not the ID number)?

Reply:
Ditto.